### PR TITLE
Vd user profile

### PR DESCRIPTION
**Pull request by viviancreates**

### CODE REVIEW

File: app/views/users/show.html.erb
Comment (merrillhuang) [inline app/views/users/show.html.erb:1]: User profile page working and displays own_photo cards correctly.

File: app/controllers/comments_controller.rb
Comment (merrillhuang) [inline app/controllers/comments_controller.rb:1]: Adding comments works and redirects to profile page.

**Review by merrillhuang — APPROVED** [review]
Everything looks good. You may want to change the root page from the Photos index page.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (marywebby) [inline app/controllers/comments_controller.rb:29]: nice job implementing `redirect_back` !

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
 
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
```
Comment (marywebby) [inline app/views/comments/_form.html.erb:16]: good refactoring on the form!

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
+                </div>
+                <div class="flex-grow-1 ms-3">
+                  <h5 class="mt-0"><%= comment.author.username %></h5>
+                  <p><%= comment.body %></p>
+                </div>
+              </div>
+            </li>
+          <% end %>
```
Comment (marywebby) [inline app/views/users/show.html.erb:29]: Nice job showcasing the comments!

File: config/routes.rb [hunk 4]
```diff
@@ -7,7 +7,9 @@
   resources :follow_requests
   resources :likes
   resources :photos
-  
+
```
Comment (marywebby) [inline config/routes.rb:9]: Don't forget to devise for users! I went and saw it was already in your code but we need to see that code in this PR

**Review by marywebby — APPROVED** [review]
Nice work!

**Review by AnnaVi11arrea1 — APPROVED** [review]
looks good! However i did run into an error when trying to sign up. 
Screenshot 2024-10-21 at 2 29 29 PM

**Pull request merged**